### PR TITLE
docs: suggest biome migrate commands after biome init

### PR DESCRIPTION
**Pull request by Conaclos**
## Summary

When `biome init` is executed the following message is printed:

```
❯ ./biome init

Welcome to Biome! Let's get you started...

Files created ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  - biome.json: Your project configuration. Documentation: https://biomejs.dev/reference/configuration

Next Steps ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  1. Setup an editor extension
     Get live errors as you type and format when you save. Learn more: https://biomejs.dev/guides/getting-started#editor-setup

  2. Try a command
     biome check verifies formatting and lint. Run biome --help for a full list of commands and options.

  3. Read the documentation
     Our website serves as a comprehensive source of guides and documentation: https://biomejs.dev

  4. Get involved in the community
     Ask questions, get support, or contribute by participating on GitHub (https://github.com/biomejs/biome),
     or join our community Discord (https://discord.gg/BypW39g6Yc)
❯ 
```

This PR:
- Add a new item `Migrate from ESLint and Prettier`
- Fix the hyperlink for the editor extension
- Rephrase and add newlines to avoid line wrapping on small screen or in dual windows mode
- Remove horizontal bars and inverse color: this looks cleaner and avoid line wrapping
- A blank line at the end

Here is the result:

```
❯ ./biome init

Welcome to Biome! Let's get you started...

Files created 

  - biome.json
    Your project configuration. See https://biomejs.dev/reference/configuration

Next Steps 

  1. Setup an editor extension
     Get live errors as you type and format when you save.
     Learn more at https://biomejs.dev/guides/integrate-in-editor/

  2. Try a command
     biome check  checks formatting, import sorting, and lint rules.
     biome --help displays the available commands.

  3. Migrate from ESLint and Prettier
     biome migrate eslint   migrates your ESLint configuration to Biome.
     biome migrate prettier migrates your Prettier configuration to Biome.

  4. Read the documentation
     Find guides and documentation at https://biomejs.dev/guides/getting-started/

  5. Get involved with the community
     Ask questions and contribute on GitHub: https://github.com/biomejs/biome
     Seek for help on Discord: https://discord.gg/BypW39g6Yc

❯ 
```

## Test Plan

CI should pass. I manually tested the command.

### CODE REVIEW

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap [hunk 1]
```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
...
   2. Try a command
-     biome ci checks for lint errors and verifies formatting. Run biome --help for a full list of commands and options.
+     biome check verifies formatting and lints.
+     biome --help displays the available commands.
 
-  3. Read the documentation
-     Our website serves as a comprehensive source of guides and documentation: https://biomejs.dev
+  3. Migrate from ESLint and Prettier
+     biome migrate eslint attempts to migrate your ESLint configuration to Biome.
+     biome migrate prettier attempts to migrate your Prettier configuration to Biome.
```
Comment (ematipico) [inline crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:44]: We should have a resource (website page) that explains this better. A guide, maybe. Not a blocker but something we should prepare, I think.

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap [hunk 2]
```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
...
 Next Steps ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   1. Setup an editor extension
-     Get live errors as you type and format when you save. Learn more: https://biomejs.dev/guides/getting-started#editor-setup
+     Get live errors as you type and format when you save.
+     Learn more: https://biomejs.dev/guides/integrate-in-editor/
 
   2. Try a command
-     biome ci checks for lint errors and verifies formatting. Run biome --help for a full list of commands and options.
+     biome check verifies formatting and lints.
```
Comment (ematipico) [inline crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:39]: My suggestion: verifies that formatting, import statements are sorted and lint rules are respected.

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap [hunk 3]
```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
...
+  3. Migrate from ESLint and Prettier
+     biome migrate eslint attempts to migrate your ESLint configuration to Biome.
+     biome migrate prettier attempts to migrate your Prettier configuration to Biome.
 
-  4. Get involved in the community
-     Ask questions, get support, or contribute by participating on GitHub (https://github.com/biomejs/biome),
-     or join our community Discord (https://discord.gg/BypW39g6Yc)
-```
+  4. Read the documentation
+     Our website provides guides and documentation: https://biomejs.dev
```
Comment (ematipico) [inline crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:47]: I think we should link the home page of the getting started page, not the home page. Mainly due to the previous phrase.

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap [hunk 4]
```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
...
 
+  5. Get involved in the community
```
Comment (ematipico) [inline crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:49]:
```suggestion
  5. Get involved with the community
```

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap [hunk 5]
```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
...
+     Ask questions and contribute on GitHub (https://github.com/biomejs/biome).
+     Join our Discord community (https://discord.gg/BypW39g6Yc).
```
Comment (ematipico) [inline crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:51]:
```suggestion
     Seek for help on Discord (https://discord.gg/BypW39g6Yc).
```

We already used the word "community", it's repetition

File: crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap (hunk 1 above)
Comment (Conaclos) [reply crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap:44]: Yeah I was thinking the sane.

**Pull request merged**